composer.json

```diff
@@ -3,7 +3,7 @@
     "license": "mit",
     "type": "statamic-addon",
     "require": {
-        "php": "^8.2",
+        "php": "^8.0.2||^8.1||^8.2",
```

Comment:
This can be ^8.1 I  believe, I'm not keen on support a PHP version that's not supported anymore.

Reply:
Understandable 😄 but you are aware that you added support for 8.0 in https://github.com/edalzell/statamic-forma/pull/17, right?